SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -261,6 +261,7 @@ void CNEMONSSolver::Viscous_Residual(CGeometry *geometry,
     numerics->SetdTvedU(nodes->GetdTvedU(iPoint), nodes->GetdTvedU(jPoint));
     numerics->SetEve   (nodes->GetEve(iPoint),    nodes->GetEve(jPoint));
     numerics->SetCvve  (nodes->GetCvve(iPoint),   nodes->GetCvve(jPoint));
+    numerics->SetGamma (nodes->GetGamma(iPoint),  nodes->GetGamma(jPoint));
```

Comment:
same comment as before, I don't think gamma is necessary for viscous fluxes

Reply:
Yup!